src/blame_git.c

```diff
@@ -478,21 +478,24 @@ static git_blame__origin* find_origin(
  * The blobs of origin and porigin exactly match, so everything origin is
  * suspected for can be blamed on the parent.
  */
-static void pass_whole_blame(git_blame *blame,
+static int pass_whole_blame(git_blame *blame,
 		git_blame__origin *origin, git_blame__origin *porigin)
 {
 	git_blame__entry *e;
 
-	if (!porigin->blob)
-		git_object_lookup((git_object**)&porigin->blob, blame->repository,
-				git_blob_id(origin->blob), GIT_OBJ_BLOB);
+	if (!porigin->blob
+	    && git_object_lookup((git_object**)&porigin->blob, blame->repository,
```

Comment:
Can you pull the `&&` up so that it's at the end of the first line, not starting the second?